### PR TITLE
Map props

### PR DESCRIPTION
**Pull request by migueloller**

### CODE REVIEW

File: packages/runtime/src/components/builtin/Carousel/Carousel.tsx [hunk 1]
```diff
@@ -72,19 +72,18 @@ type Props = {
...
   slideBorder?: BorderValue
   slideBorderRadius?: BorderRadiusValue
 }
 
-const Wrapper = styled.div<{ width: Props['width']; margin: Props['margin'] }>`
+const Wrapper = styled.div<{ margin: Props['margin'] }>`
   position: relative;
   display: flex;
   flex-direction: column;
-  ${cssWidth('400px')}
```
Comment (apledger) [inline packages/runtime/src/components/builtin/Carousel/Carousel.tsx:87]: Where did the defaulted `400px` value go?

File: packages/runtime/src/components/builtin/Countdown/Countdown.tsx [hunk 2]
```diff
@@ -71,7 +69,6 @@ const Container = styled.div<{
   labelFont?: ResponsiveValue<string>
 }>`
   display: flex;
-  ${cssWidth('560px')}
```
Comment (apledger) [inline packages/runtime/src/components/builtin/Countdown/Countdown.tsx:74]: And this `560px`?

File: packages/runtime/src/components/builtin/Video/Video.tsx [hunk 3]
```diff
@@ -1,35 +1,31 @@
...
-const Container = styled.div<{
-  width: Props['width']
-  margin: Props['margin']
-  borderRadius: Props['borderRadius']
-}>`
+const Container = styled.div<{ margin: Props['margin']; borderRadius: Props['borderRadius'] }>`
   display: flex;
   flex-direction: column;
   overflow: hidden;
-  ${cssWidth('560px')}
```
Comment (apledger) [inline packages/runtime/src/components/builtin/Video/Video.tsx:32]: Where did this `560px` go?

**Review by apledger — COMMENTED** [review]
Looks good to me! Just a few questions about the default widths we were using in some components.

File: packages/runtime/src/components/builtin/Countdown/Countdown.tsx (hunk 2 above)
Comment (migueloller) [reply packages/runtime/src/components/builtin/Countdown/Countdown.tsx:74]: It's inferred automatically from the control's default value 😍

File: packages/runtime/src/components/builtin/Video/Video.tsx (hunk 3 above)
Comment (migueloller) [reply packages/runtime/src/components/builtin/Video/Video.tsx:32]: https://github.com/makeswift/makeswift/pull/39#discussion_r852432673

File: packages/runtime/src/components/builtin/Carousel/Carousel.tsx (hunk 1 above)
Comment (migueloller) [reply packages/runtime/src/components/builtin/Carousel/Carousel.tsx:87]: https://github.com/makeswift/makeswift/pull/39#discussion_r852432673

**Comment by migueloller** [comment]
> Looks good to me! Just a few questions about the default widths we were using in some components.

The values are determined automatically from the control's default value. For example:

```typescript
Controls.Width({ defaultValue: { value: 560, unit: 'px' } })
```

Will automatically apply that.

**Pull request merged**